app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,20 @@
+class RelationshipsController < ApplicationController
+  before_filter :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.delete if relationship.follower == current_user
```

Comment:
Be careful using [delete](http://apidock.com/rails/ActiveRecord/Base/delete/class). As you can read there, callbacks are not executed, which can cause you hard-to-diagnose bugs. [delete](http://apidock.com/rails/ActiveRecord/Base/destroy/class) is the usual method which does execute the callbacks.